pkg/agent/flowexporter/types.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package flowexporter
+
+import (
+	"net"
+	"time"
+)
+
+const (
+	PollInterval = 5 * time.Second
+)
+
+type ConnectionKey [5]string
+
+type Tuple struct {
+	SourceAddress      net.IP
+	DestinationAddress net.IP
+	Protocol           uint8
+	SourcePort         uint16
+	DestinationPort    uint16
+}
+
+type Connection struct {
+	// Fields from conntrack flows
+	ID                             uint32
+	Timeout                        uint32
+	StartTime                      time.Time
+	StopTime                       time.Time
```

Comment:
can we add a comment explaining what `StopTime` is, because I am not sure myself?

Reply:
Honestly, I did not do complete experimenting with stop timestamp as it was not used for any visualization aspect yet. This is my understanding:
For invalid and closed connections: stop time is the time when connection was updated last.
For established connections: stop time is latest time when it was polled.

Adding above for now.